nexus/test-utils-macros/src/lib.rs

```diff
@@ -3,33 +3,42 @@ use quote::quote;
 use std::collections::HashSet as Set;
 use syn::parse::{Parse, ParseStream, Result};
 use syn::punctuated::Punctuated;
+use syn::spanned::Spanned;
 use syn::{parse_macro_input, ItemFn, Token};
 
 #[derive(Debug, PartialEq, Eq, Hash)]
-pub(crate) struct NameValue {
-    name: syn::Path,
-    _eq_token: syn::token::Eq,
-    value: syn::Path,
+pub(crate) enum NexusTestArg {
```

Comment:
Nice: I like this structure much better.

Reply:
Thanks :)